src/sentry/static/sentry/app/components/fileChange.tsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {AvatarUser} from 'app/types';
+import AvatarList from 'app/components/avatar/avatarList';
+import {IconFile} from 'app/icons';
+
+type Props = {
+  filename: string;
+  authors: AvatarUser[];
+};
+
+function FileChange({filename, authors}: Props) {
+  return (
+    <FileItem className="list-group-item list-group-item-sm">
```

Comment:
Since `FileItem` is a styled component, should we dupe the `list-group-item` styles here so that we can remove the CSS class name dependency?

Reply:
Yeah I can do that.